Widgets/Value.php

```diff
@@ -28,21 +28,21 @@
 
```

Comment:
Kannst du bitte versuchen die Zeilenenden nochmal in UNIX umzuwandeln, damit ein Diff sinnvoll möglich ist?